src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -67,7 +67,11 @@ def _splitname_parser(line):
 
     :returns: dictionary with stripped prefixes
     """
-    parts = line[0].split(';')
+    if len(lines) != 1:
```

Comment:
when debugging this, it would be very helpful to see the number of lines that it got instead of 1